lib/jsdom/level2/html.js

```diff
@@ -2102,6 +2121,16 @@ define('HTMLFrameElement', {
               self.removeEventListener('DOMNodeInsertedIntoDocument', loader, false);
               this._waitingOnInsert = false;
               loadFrame(self);
+
+              parent._length++;
+              refreshAccessors(self._ownerDocument);
+
+              var name = self.getAttribute("name");
```

Comment:
I checked, apparently the following two lines are different when executed at the same place
```js
defineGetter(window, i, function () { return frame.contentWindow; });
window._frame(i, frame);
```

No idea why that is, apparently the getter defined in _frame is never called. Maybe it's once again the proxy thing?